components/scheduler/views/month.md

```diff
@@ -10,7 +10,7 @@ position: 4
 
 # Month View
 
-The Month view of the Scheduler for Blazor shows an entire month to the user.
+The Month view of the Scheduler for Blazor shows an entire month to the user. Also the Month view can be configured to display the events that are [grouped by resource]({%slug scheduler-resource-grouping%}).
```

Comment:
Apply my notes about active voice as needed throughout. I would not be marking any more instances.